test/unit/inspec_tools/metadata_test.rb

```diff
@@ -0,0 +1,23 @@
+# This command is to be used by yes|inspec_tools generate_ckl_metadata in the build.yml for github.
```

Comment:
If you are having trouble with standard in, instead of expecting the input to check to come into the program as standard input, take a filename and read the file or just use a constant filename for testing purposes?